images/wordpress/wp-start

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+if test ${WORDPRESS_VERSION}X = latestX; then
+  wp core download --path=/var/www/html
+else
```

Comment:
And this was also based in my docker misconceptions. Sorry. This is all good to go.